src/pages/NewChatPage.js

```diff
@@ -186,7 +193,14 @@ function NewChatPage(props) {
         >
             {({didScreenTransitionEnd, safeAreaPaddingBottomStyle}) => (
                 <>
-                    <HeaderWithBackButton title={props.isGroupChat ? props.translate('sidebarScreen.newGroup') : props.translate('sidebarScreen.newChat')} />
+                    <HeaderWithBackButton
+                        title={
+                            share ? props.translate('newChatPage.shareToExpensify') : props.isGroupChat ? props.translate('sidebarScreen.newGroup') : props.translate('sidebarScreen.newChat')
+                        }
+                        shouldShowBackButton={!share}
```

Comment:
LOL this is kinda just funny

Reply:
![](https://media.giphy.com/media/zGmilchBguJeU/giphy.gif)